### PR TITLE
add single checkbox to the form builder

### PR DESCRIPTION
**Pull request by cnorthwood**
This is used to e.g., record consent for processing special purpose data

### CODE REVIEW

**Review by KPobeeNorris — COMMENTED** [review]
I've not really had much to do with this repo, but the code changes look good to me.  I just don't know if you'd want another quick pair of eyes on it from someone like Leeky who's done more work here?

**Comment by cnorthwood** [comment]
that'd be good if you could cast your eyes over it @leeky as more familiar?

File: spec/features/check_box_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  it "renders the visible unchecked checkbox and label" do
+    expect(page).to have_unchecked_field("person[opted_in]", type: "checkbox", with: "1")
+  end
+
+  it "renders the hidden field which defaults to false" do
+    expect(page).to have_field("person[opted_in]", type: "hidden", with: "0")
+  end
+
+  it "renders the visible checked checkbox and label" do
```
Comment (leeky) [inline spec/features/check_box_spec.rb]: The description mentions the label, however the test body doesn't check for one. You can target the checkbox via the label text with:

```ruby
expect(page).to have_checked_field("Account active", type: "checkbox", with: "1")
```

File: lib/citizens_advice_form_builder/elements.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@
 require_relative "elements/text_area"
 require_relative "elements/date_input"
 require_relative "elements/error_summary"
+require_relative "elements/check_box"
```
Comment (leeky) [inline lib/citizens_advice_form_builder/elements.rb:9]: You should `require` the relevant design system component in `lib/citizens_advice_form_builder/form_builder.rb` too.

**Review by leeky — APPROVED** [review]
LGTM! :shipit: 

Thank you for adding this! Just a couple of small tweaks needed.

**Pull request merged**